### PR TITLE
Historic agg v2 support different date types

### PR DESCRIPTION
**Pull request by shlomikushchi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -294,7 +294,13 @@ The example below gives AAPL daily OHLCV data in a DataFrame format.
 import alpaca_trade_api as tradeapi
 
 api = tradeapi.REST()
+# all of these examples work
 aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from='2019-01-01', to='2019-02-01').df
+aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from=dateutil.parser.parse('2019-01-01'), to='2019-01-10').df
+aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from=dateutil.parser.parse('2019-01-01').date(), to='2019-01-10').df
```
Comment (umitanuki) [inline README.md:300]: As an example/document, it's a bit confusing to put this conversion? How about `datetime.date(2019, 1, 1)`?

I couldn't tell what type this is `date()` was returning but maybe only me.

File: README.md [hunk 2]
```diff
@@ -294,7 +294,13 @@ The example below gives AAPL daily OHLCV data in a DataFrame format.
...
+aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from=pd.Timestamp('2019-01-01'), to='2019-01-10').df
+# timestamp should be in milliseconds
+aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from=dateutil.parser.parse('2019-01-01').timestamp()*1000, to='2019-01-10').df
```
Comment (umitanuki) [inline README.md:303]: Also I would put `1546329600000` instead of dateutil...

File: README.md [hunk 3]
```diff
@@ -294,7 +294,13 @@ The example below gives AAPL daily OHLCV data in a DataFrame format.
 import alpaca_trade_api as tradeapi
 
 api = tradeapi.REST()
+# all of these examples work
 aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from='2019-01-01', to='2019-02-01').df
+aapl = api.polygon.historic_agg_v2('AAPL', 1, 'day', _from=dateutil.parser.parse('2019-01-01'), to='2019-01-10').df
```
Comment (umitanuki) [inline README.md:299]: we should keep it consistent to use to='2019-02-01' with existing example.

File: README.md [hunk 4]
```diff
@@ -335,8 +341,11 @@ object.
 - `multiplier` is an integer affecting the amount of data contained in each Agg object.
 - `timespan` is a string affecting the length of time represented by each Agg object. It is one of the following values:
   - `minute`, `hour`, `day`, `week`, `month`, `quarter`, `year`
-- `_from` is an Eastern Time timestamp string that filters the result for the lower bound, inclusive.
-- `to` is an Eastern Time timestamp string that filters the result for the upper bound, inclusive.
+- `_from` is an Eastern Time timestamp string/object that filters the result
+ for the lower bound, inclusive. we accept the date in these formats: 
+ datetime.datetime, datetime.date, pd.Timestamp, datetime.timestamp(),
```
Comment (umitanuki) [inline README.md:346]: Why only `timestamp` has parenthesis?

File: README.md (hunk 2 above)
Comment (shlomikushchi) [reply README.md:303]: sure. I just wanted to show that we start always from the same thing. but sure

File: README.md (hunk 3 above)
Comment (shlomikushchi) [reply README.md:299]: fixed

File: README.md (hunk 4 above)
Comment (shlomikushchi) [reply README.md:346]: fixed

**Pull request merged**